vita3k/gui/src/gui.cpp

```diff
@@ -224,14 +224,25 @@ static void init_font(GuiState &gui, EmuEnvState &emuenv) {
 
         gui.vita_font = io.Fonts->AddFontFromFileTTF(fs_utils::path_to_utf8(latin_fw_font_path).c_str(), font_config.SizePixels, &font_config, latin_range);
         font_config.MergeMode = true;
+        bool has_added_asia_font = false; // To reduce conflicts between CJKV ideograms
+
+        // Add fonts for system language first
+        const auto sys_lang = static_cast<SceSystemParamLang>(emuenv.cfg.sys_lang);
+        if (sys_lang == SCE_SYSTEM_PARAM_LANG_KOREAN) {
+            io.Fonts->AddFontFromFileTTF(fs_utils::path_to_utf8(fw_font_path / "kr0.pvf").c_str(), font_config.SizePixels, &font_config, korean_range);
+            has_added_asia_font = true;
+        } else if (sys_lang == SCE_SYSTEM_PARAM_LANG_CHINESE_T || sys_lang == SCE_SYSTEM_PARAM_LANG_CHINESE_S) {
+            io.Fonts->AddFontFromFileTTF(fs_utils::path_to_utf8(fw_font_path / "cn0.pvf").c_str(), font_config.SizePixels, &font_config, chinese_range);
+            has_added_asia_font = true;
+        }
 
         io.Fonts->AddFontFromFileTTF(fs_utils::path_to_utf8(fw_font_path / "jpn0.pvf").c_str(), font_config.SizePixels, &font_config, japanese_and_extra_ranges.Data);
 
-        const auto sys_lang = static_cast<SceSystemParamLang>(emuenv.cfg.sys_lang);
-        if (emuenv.cfg.asia_font_support || (sys_lang == SCE_SYSTEM_PARAM_LANG_KOREAN))
+        // Add Asian fonts finally to avoid conflict with Japanese
```

Comment:
i no understund, you create two time same code, but doing diferent things, asia support want litteraly said, load koran AND chinese, so no have sense here

Reply:
again, here you no load att all korean if you load chinese, is worng
and same if you load k,orean missing chinese for asia font support